dataAccessObjects.py

```diff
@@ -283,6 +283,19 @@ def getImgSystemName(self,imgId):
         sql = 'SELECT img_system_name FROM image_data WHERE img_id="{imgId}"'.format(imgId=imgId)
         return self.queryOneValue(sql)
 
+    def insertImgData(self,imgData):
+        sql = 'INSERT INTO image_data ' \
+            + '(img_id, type_id, img_system_name, img_thumbnail_name, img_file_name, img_start_date, ' \
+            + 'img_end_date, img_start_time, img_end_time, img_display_time, user_id) ' \
+            + ' VALUES ' \
+            + '("{id}", {typeId}, "{systemName}", "{thumbnailName}", "{fileName}", "{startDate}",' \
+            + ' "{endDate}", "{startTime}", "{endTime}", {displayTime}, {userId})'
+        sql.format(id=imgData["id"],typeId=imgData["type_id"],systemName=imgData["system_name"], 
+                thumbnailName=imgData["thumbnail_name"],fileName=imgData["file_name"],startDate=imgData["start_date"],
+                endDate=imgData["end_date"],startTime=imgData["start_time"],endTime=imgData["end_time"], 
+                displayTime=imgData["display_time"],userId=imgData["user_id"])
+        self.db.cmd(sql)
```

Comment:
Did you try it?
I use python online compiler to test the sql string.
It was not correct. It could be online compiler problem or real problem.
Please check it twice. Thanks !